app/main.py

```diff
@@ -1,3 +1,66 @@
-class Distance:
-    # Write your code here
-    pass
+class Distance:
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __add__(self, other: callable) -> callable:

```

Comment:
other is not callable, other can be int, float or Distance, and you return Distance. Change here and other places